test/testcafe/spec/IdentifyWithPassword_spec.js

```diff
@@ -148,22 +148,6 @@ test.meta('gen3', false).requestHooks(identifyRequestLogger, identifyWithPasswor
   await t.expect(await identityPage.hasShowTogglePasswordIcon()).notOk();
 });
 
-test.requestHooks(identifyRequestLogger, identifyWithPasswordMock)('should auto-hide password after 30 seconds', async t => {
```

Comment:
Maybe just disable this test for gen3 only? Gen1/2 should support this feature

Reply:
Yep, I should've done this. Not sure why I fully deleted it